### ISSUE TITLE
Reason for [PAD] class

### ISSUE THREAD

**Issue by polars05** [BrikerMan/Kashgari#135]
The various classification models (CNN_Model, CNN_LSTM_Model, etc.) seem to introduce a [PAD] class, resulting in +1 more labels (e.g. if I want to classify between "ham" and "spam", the final dense layer is of size 3)

I believe the relevant section of the code is here: https://github.com/BrikerMan/Kashgari/blob/6f3188b0c88d98f18ac0aeeb071e8cbcd7deade0/kashgari/tasks/classification/base_model.py#L142-L145

Is there a reason for introducing the [PAD] class? Thanks!

**Comment by alexwwang** [comment]
Yes and it won't make your model prediction less accurate, I assume.

**Comment by BrikerMan** [comment]
This code is originally for the sequence labeling padding, it does useless and no need to keep in the classification task. It has been removed in the future tf.keras branch.

**Comment by polars05** [comment]
Thanks for pointing me to the tf.keras branch!
